neonvm-kernel/Dockerfile

```diff
@@ -25,7 +25,26 @@ RUN apt-get update && apt-get -y install \
     python3 \
     cpio \
     zstd \
-    libncurses-dev
+    libncurses-dev \
```

Comment:
should we introduce a separate build phase to make cleaner dependency list?

Reply:
Done. Please re-review.